config/migrations/2025/20250224124500-draft-to-niet-bekrachtigd-for-non-iv-mandatarissen.sparql

```diff
@@ -0,0 +1,40 @@
+prefix org: <http://www.w3.org/ns/org#>
+prefix lmb: <http://lblod.data.gift/vocabularies/lmb/>
+prefix ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX dct: <http://purl.org/dc/terms/>
+
+DELETE {
+  GRAPH ?g {
+    ?mandataris lmb:hasPublicationStatus ?draftStatus .
+    ?mandataris dct:modified ?modified .
+  }
+}
+INSERT {
+  GRAPH ?g {
+    ?mandataris lmb:hasPublicationStatus <http://data.lblod.info/id/concept/MandatarisPublicationStatusCode/d3b12468-3720-4cb0-95b4-6aa2996ab188> . # Niet bekrachtigd formal effectief status
+    ?mandataris dct:modified ?now .
+  }
+}
+WHERE {
+  {
+    GRAPH ?g {
+      ?mandataris lmb:hasPublicationStatus <http://data.lblod.info/id/concept/MandatarisPublicationStatusCode/588ce330-4abb-4448-9776-a17d9305df07> . # Draft publication status
+      ?mandataris org:holds / ^org:hasPost ?boi .
+      ?boi lmb:heeftBestuursperiode ?bestuursperiode .
+      FILTER(?bestuursperiode != <http://data.lblod.info/id/concept/Bestuursperiode/96efb929-5d83-48fa-bfbb-b98dfb1180c7>) # All other periods than 2024 - heden
+    }
+    ?g ext:ownedBy ?eenheid .
+  } UNION {
+    SELECT DISTINCT ?mandataris
+    WHERE {
+      GRAPH ?g {
+        ?mandataris lmb:hasPublicationStatus <http://data.lblod.info/id/concept/MandatarisPublicationStatusCode/588ce330-4abb-4448-9776-a17d9305df07> . # Draft publication status
+        ?mandataris org:holds / ^org:hasPost ?boi .
+        ?boi lmb:heeftBestuursperiode <http://data.lblod.info/id/concept/Bestuursperiode/96efb929-5d83-48fa-bfbb-b98dfb1180c7> . #2024 - heden
+
+        ?eenheid ext:voorbereidingVerborgen ?isIvFinished . # IV must be finished
```

Comment:
better to check the status of the iv because meanwhile at least one extra was finished